addon/components/scheduled-banner/component.js

```diff
@@ -54,17 +54,19 @@ export default Ember.Component.extend(Analytics, {
                 crossDomain: true
             })
             .then(({data: { attributes: attrs, links}}) => {
-                this.setProperties({
-                    color: attrs.color,
-                    defaultAltText: attrs.default_alt_text,
-                    mobileAltText: attrs.mobile_alt_text,
-                    defaultPhoto: links.default_photo,
-                    mobilePhoto: links.mobile_photo,
-                    startDate: attrs.start_date,
-                    link: attrs.link,
-                    name: attrs.name,
-                });
-                this.get('startDate') && this.set('hidden', false);
+                if (attrs.start_date) {
```

Comment:
Is this commit related to this ticket?

Reply:
It was to fix some failing tests. I will use currentUser stub as well.